proxy/docker/Dockerfile-for-release

```diff
@@ -17,6 +17,7 @@ RUN tdnf install -y \
 RUN /usr/sbin/groupadd -g 2000 wavefront
 RUN useradd --comment '' --uid 1000 --gid 2000 wavefront
 RUN chown -R wavefront:wavefront /var
+RUN chown -R wavefront:wavefront /usr/lib/jvm/OpenJDK-1.11.0/lib/security/cacerts
```

Comment:
We can just move this to line number 19 and combine the 2 steps to 1.

```
RUN chown -R wavefront:wavefront /var /usr/lib/jvm
```

Also do we need `chmod` as well?